source/clear-linux/guides/maintenance/download-verify-decompress.rst

```diff
@@ -0,0 +1,180 @@
+.. _download-verify-decompress-linux:
+.. _download-verify-decompress-mac:
+.. _download-verify-decompress-windows:
+
+
+Download, verify, and decompress a |CL-ATTR| image
+##################################################
+
+This guide describes the types of |CL| images available, where to download
+them, how to verify the integrity of an image, and how to decompress it.
+
+
```

Comment:
Would you add a local table of contents right after the intro sentence? Will make this standard w/ other guide updates. 

..contents:: 
  :local:
  :depth: 1